src/Libraries/DSOffice/Excel.cs

```diff
@@ -270,6 +270,7 @@ public static WorkBook SaveAsExcelWorkbook(WorkBook workbook, string filename)
         /// <param name="readAsStrings">toggle to switch between reading Excel file as strings only or not</param>
         /// <returns name="data">Rows of data from the Excel worksheet.</returns>
         /// <search>office,excel,spreadsheet,ifequalreturnindex</search>
+        [IsVisibleInDynamoLibrary(false)]
```

Comment:
You don't need to mark these nodes hidden in library view because this DLL should not be loaded in Library. Make sure to remove the registration of this DLL from host applications.